### PR TITLE
Display outlier information from MSG_INS_UPDATES in Swift console [INFRA-383]

### PR DESCRIPTION
**Pull request by notoriaga**

### CODE REVIEW

**Comment by silverjam** [comment]
@notoriaga got any test data I can use with str2str to see the lights twinkle?

**Comment by notoriaga** [comment]
> @notoriaga got any test data I can use with str2str to see the lights twinkle?

`drive-test-logs-2/2021-02/2021-02-11-WheelODORoute/PGM2_CAN_v1_2_34_wheelticks_OSR_staging_10Hz_AmotechL5/11-185112.sbp` (and `.tag`) is the one I've been using

**Comment by silverjam** [comment]
How hard would it be to use a unicode character and color it differently, something like https://www.fileformat.info/info/unicode/char/25cf/index.htm?  Layout looks a little odd on Windows:

Screenshot 2021-02-26 191718

**Comment by notoriaga** [comment]
> How hard would it be to use a unicode character and color it differently, something like ●: https://www.fileformat.info/info/unicode/char/25cf/index.htm? Layout looks a little odd on Windows:
> 
> Screenshot 2021-02-26 191718

Hmm I tried that but couldn't find a way to change the text color afterwards. And the unicode chars with color don't seem to be supported

**Comment by silverjam** [comment]
@notoriaga I got something working here: https://github.com/swift-nav/piksi_tools/pull/1227

**Comment by silverjam** [comment]
> Hmm I tried that but couldn't find a way to change the text color afterwards. And the unicode chars with color don't seem to be supported

Hrmm, maybe I need to test on all platforms then, maybe it's something that works on Windows, but not Linux?

**Comment by notoriaga** [comment]
> > Hmm I tried that but couldn't find a way to change the text color afterwards. And the unicode chars with color don't seem to be supported
> 
> Hrmm, maybe I need to test on all platforms then, maybe it's something that works on Windows, but not Linux?

What you have seems to work on linux too, I was trying to use extended unicode characters like - https://unicode-table.com/en/1F534/

**Review by silverjam — APPROVED** [review]

File: piksi_tools/console/fusion_engine_status.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+    else:
+        return '* { color: grey; }'
+
+
+def check_flag(flag):
+    return lambda msg: (getattr(msg, flag) & 0b1111) == 0
+
+
+def stats_key(flag):
+    return f'stats_{flag}'
```
Comment (silverjam) [inline piksi_tools/console/fusion_engine_status.py:78]: We're using Python 3.5 for deploying currently 😬 -- so need to transform this to "old style" string interpolation

**Review by silverjam — APPROVED** [review]
lgtm, need to fix-up the f-strings then we're good

**Comment by silverjam** [comment]
@denniszollo any feedback?

**Comment by notoriaga** [comment]
CI seems to be running into some docker hub rate limit

**Comment by silverjam** [comment]
@notoriaga still hitting some throttling but I re-kicked and it seems to have gotten past the docker pull step for now

**Review by denniszollo — APPROVED** [review]
Looks great!  I've done some limited testing and reviewed the logic and I think it behaves as expected and should help us understand that data is or is/not being used by fusion engine.

**Pull request merged**